### ISSUE TITLE
Nan in summary histogram when running facenet_train_classifier.py

### ISSUE THREAD

**Issue by YuliyaLi** [davidsandberg/facenet#162]
I have the following problem when i use my own dataset to run  facenet_train_classifier.py. All parameters are the same except the dataset path: 

Epoch: [0][1/100]	Time 4.541	Loss 44.833	RegLoss 24.607
Epoch: [0][2/100]	Time 2.765	Loss 46.207	RegLoss 24.736
Epoch: [0][3/100]	Time 0.412	Loss 45.358	RegLoss 24.623
Epoch: [0][4/100]	Time 0.412	Loss 46.909	RegLoss 24.945
Epoch: [0][5/100]	Time 0.411	Loss 43.843	RegLoss 24.535
.....
InvalidArgumentError (see above for traceback): Nan in summary histogram for: Logits/biases/gradients
	 [[Node: Logits/biases/gradients = HistogramSummary[T=DT_FLOAT, _device="/job:localhost/replica:0/task:0/cpu:0"](Logits/biases/gradients/tag, gradients/Logits/BiasAdd_grad/tuple/control_dependency_1/_2081)]]

Anybody can solve this problem? many thanks!

**Comment by davidsandberg** [comment]
For some reason the gradients become NaN. It's difficult to say what's the reason for this, but in general it could be due to that the training becomes instable du to too high learning rate etc. But in this case I don't know.
